README.md

```diff
@@ -19,10 +22,12 @@ Options:
   --margin-left                                              [default: "6.25mm"]
   --format                                                   [default: "Letter"]
   --landscape                                         [boolean] [default: false]
+  --viewport-width                                    [number] [default: 0]
+  --viewport-height                                    [number] [default: 0]
```

Comment:
@fingul what would you think of making this one option in the format `--viewport 800x600`?

I feel like it doesn't make sense to set either of these on their own, so rather than protecting against that how about just make it impossible by making them one logical option?

You might apply some logic such than empty = auto, one number = square, two numbers = rectangle